src/python/test/test_dxclient.py

```diff
@@ -7489,6 +7489,43 @@ def test_get_app_omit_resources(self):
                     break
             self.assertTrue(seenResources)
 
+    @unittest.skipUnless(testutil.TEST_ENV, 'skipping test that would clobber your local environment')
+    @unittest.skipUnless(testutil.TEST_ISOLATED_ENV, 'skipping test that would create apps')
+    @unittest.skipUnless(testutil.TEST_MULTIPLE_USERS, 'skipping test that would require another user')
```

Comment:
from @ptn24: Note that we can combine these into something like:
`@unittest.skipUnless(FOO and BAR, 'skipping test that would foo and bar')`